libs/shared/rule/result/src/rule-result.helpers.spec.ts

```diff
@@ -284,7 +287,7 @@ describe('signRequest', () => {
     } as never);
 
     await expect(signRequest(input)).rejects.toThrow(
-      'Error on typia.assert(): invalid type on $input, expect to be Credentials',
+      'Error on createAssert(): invalid type on $input, expect to be Credentials',
```

Comment:
In palantir, I opted by changing the expected error messages to remove the first part, so it is not affected if we interchange between assert, createAssert, assertEquals and so on:
```suggestion
      'invalid type on $input, expect to be Credentials',
```